src/main/frontend/fs.cljs

```diff
@@ -74,9 +74,9 @@
   [repo dir path content opts]
   (when content
     (let [fs-record (get-fs dir)]
-      (p/let [metadata-or-css? (or (string/ends-with? path (str "/" config/metadata-file))
-                                   (string/ends-with? path (str "/" config/custom-css-file)))
-              content (if metadata-or-css? content (encrypt/encrypt content))]
+      (p/let [md-or-org? (or (string/ends-with? path ".md")
```

Comment:
`path` might be ended with .md, .markdown, .Markdown, .ORG, we might need to lowercase the suffix and test whether it's in #{md, markdown, org}.